hax/hax/server.py

```diff
@@ -233,6 +261,7 @@ def _configure(self) -> None:
         app = self._create_server()
         app.add_routes([
             web.get('/', hello_reply),
+            web.get('/hctl_stat', hctl_stat),
```

Comment:
I think the endpoint should be more reader friendly, e.g. `GET /cluster/status`.
The output should be in JSON rather than a plaintext (because REST endpoints are normally machine-friendly because of possible integrations).

Reply:
Updated